src/utils/auto-box-collider.js

```diff
@@ -0,0 +1,20 @@
+export function getBox(entity, boxRoot) {
```

Comment:
cache rotation and copy instead of clone
```
export function getBox() {
  const rotation = new THEE.Quaternion();
  return function (entity, boxRoot) {
    ...
    rotation.copy(entity.object3D.rotation);
...
```